framework/validators/InlineValidator.php

```diff
@@ -27,11 +28,12 @@ class InlineValidator extends Validator
     /**
      * @var string|\Closure an anonymous function or the name of a model class method that will be
      * called to perform the actual validation. The signature of the method should be like the following,
-     * where `$attribute` is the name of the attribute to be validated, and `$params` contains the value
-     * of [[params]] that you specify when declaring the inline validation rule:
+     * where `$attribute` is the name of the attribute to be validated, `$params` contains the value
```

Comment:
this would be more readable in list form:
- `$attribute` is the ...
- `$params` contains ...